l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -707,7 +707,7 @@ def setAttachments(self, invoice, body):
         if invoice.fatturapa_doc_attachments:
             for doc_id in invoice.fatturapa_doc_attachments:
                 AttachDoc = AllegatiType(
-                    NomeAttachment=doc_id.datas_fname,
+                    NomeAttachment=doc_id.datas_fname[:60],
```

Comment:
@sergiocorato se non sbaglio in questo modo però verrebbe rimossa l'estensione del file